tests/test_modeling_common.py

```diff
@@ -236,11 +236,14 @@ def _create_and_check_torchscript(self, config, inputs_dict):
             loaded_model.to(torch_device)
             loaded_model.eval()
 
-            model_params = model.parameters()
-            loaded_model_params = loaded_model.parameters()
+            model_state_dict = model.state_dict()
+            loaded_model_state_dict = loaded_model.state_dict()
+
+            self.assertEqual(set(model_state_dict.keys()), set(loaded_model_state_dict.keys()))
```

Comment:
If you'd like, you can use `self.assertEqual(set(model_state_dict), set(loaded_model_state_dict))` because iterating a `dict` yields its keys. 

Adding `.keys()` is fine too if you prefer the more explicit version.